wiki/fr/fr-Installation-for-Windows.md

```diff
@@ -0,0 +1,53 @@
+*ASIO4All n'est pas recommandé si votre interface audio fournit son propre pilote ASIO, car celui-ci pourrait avoir une meilleure latence.*
+
+Tout d'abord, **fermez toutes les applications** (en particulier celles qui pourraient accéder à votre carte son comme votre navigateur) pour minimiser les conflits. Si l'audio ne fonctionne pas « sortie de la boîte », assurez-vous que seules les **entrées/sorties correctes** de son panneau de contrôle **sont activées**.  
+Vous devrez peut-être expérimenter un peu pour trouver les bonnes, car chaque ordinateur est un peu différent. Faites-le pendant que vous êtes connecté à un serveur pour entendre votre instrument ou votre voix afin de vérifier que tout soit correctement configuré.
+
+### Comment configurer les entrées ASIO4All (Guide)
+
+1. Ouvrez les paramètres de Jamulus
+1. Allez à « _ASIO Settings_ » (Configuration ASIO)
+1. Activer le panneau _advanced view_ dans ASIO4All (cliquez sur l'icône en bas à droite)
+1. Ouvrez les entrées de votre interface audio en cliquant sur l'_icône plus_ à côté du nom de l'interface.
+1. Activez/désactivez alors les entrées/sorties correctes
+
+De nombreuses cartes son internes sont appelées Realtek High Definition Audio, Conexant ou similaire.
+Les casques et les haut-parleurs sont souvent identifiés comme des « sorties » et les micros comme des « entrées ».
+
+### Vous avez des problèmes avec la configuration d'ASIO4All ?
+
+Bien qu'ASIO4All fonctionne normalement « à la sortie de la boîte », vous pourriez rencontrer des difficultés avec sa configuration.
+
+Si rien ne fonctionne, essayez tout d'abord de **redémarrer Jamulus et/ou votre ordinateur**.
+Ensuite, essayez de **configurer les entrées/sorties à nouveau**. Les entrées/sorties actives et accessibles affichent un bouton marche/arrêt allumé et un bouton de lecture. Si vous voyez une croix rouge ou un symbole jaune, vous devrez peut-être fermer d'autres applications comme votre navigateur, Zoom…
+
+Regardez cette [vidéo](https://youtu.be/_GzOsitVgLI){: target="_blank" rel="noopener noreferrer"} sur Youtube (en anglais) de Andrew Williams qui donne des informations plus détaillées sur la configuration d'ASIO4All.
+
+Des informations plus détaillées sur la façon de configurer ASIO4All sont aussi documentées dans les [FAQ officielles d'ASIO4All](http://www.asio4all.org/faq.html){: target="_blank" rel="noopener noreferrer"} sur le site web d'ASIO4All ainsi que dans le manuel d'ASIO4All. Vous trouverez très probablement ces informations sur votre bureau ou dans le dossier où le binaire d'ASIO4All est installé (généralement `C:\Program Files (x86)\ASIO4ALL v2\`).
+
+## Tout est installé ?
+Jamulus a été installé et peut désormais être utilisé. Si vous n'avez pas encore installé votre matériel audio, consultez la section [configuration du matériel](Hardware-Setup).
+
+Pour des informations plus détaillées sur l'utilisation de Jamulus, vous pouvez consulter le [Manuel du logiciel](Software-Manual).
```

Comment:
* "_vous pouvez_" -> "_veuillez_"
* "_Manuel_" -> "_manuel_" (no capital letter)

Reply:
note this ?
* "vous pouvez" -> "veuillez"